pkg/printers/printers.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The following is a subset of original implementation
+// https://github.com/kubernetes/kubernetes/blob/v1.15.0-alpha.2/pkg/printers/internalversion/printers.go
+
+package printers
+
+import (
+	"fmt"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	servingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1beta1 "k8s.io/apimachinery/pkg/apis/meta/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/duration"
+	"time"
+)
+
+// AddHandlers adds print handlers for knative resources
+func AddHandlers(h PrintHandler) {
+	kServiceColumnDefinitions := []metav1beta1.TableColumnDefinition{
```

Comment:
can we "place" kServiceColumnDefinitions closer to the service get command. it seems that pkg/printers package is meant to be generic and it shouldnt have specific knowledge of knative service columns.

Reply:
@cppforlife: I think yes, we can place it in `service_get_flags.go`. Another option is to create `internalversion` pkg in `pkg/printers/` and have `printers.go` file with columns and their printer functions (this is how its done in for kubectl). 
I think moving the columns closer to service get command make sense, we'll need to add the handlers for individual commands in `ToPrinter` method though.

The number of files in `pkg/kn/commands` is increasing, we'll soon be requiring some refactoring to have individual command's sub-packages (as its done in knctl).